envoy/api/v2/auth/local_rbac.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.api.v2.auth;
+
+import "envoy/api/v2/auth/rbac.proto";
+
+// LocalRBAC specifies Istio RBAC policies that apply to the local RBAC engine
+// that typically runs on a proxy. There can be multiple services running behind
+// a proxy.
+//
+// Here is an example of local RBAC policy, which contains policies for a single service.
+// [
+//   {
+//     service: products.default.svc.cluster.local
+//     policies:
+//     - role: service-admin
+//       spec:
+//         rules:
+//         - services: [“*”]
+//           methods: [“*”]
+//       bindings:
+//       - subjects:
+//         - user: “cluster.local/ns/default/sa/admin”
+//         roleRef:
```

Comment:
since each `Policy` holds its own `ServiceRoleBinding`s, what is the need for the `RoleRef`? Are there cases where the binding is independent of the `Policy`?

Reply:
I made roleRef an optional field in ServiceRoleBinding. This way, when used in filter config, roleRef part can be skipped. RoleRef is needed when a user defines a ServiceRoleBinding object. I updated the example here and removed "roleRef" section.

Reply:
So you're saying when the Bindings is independent of the Policy? When does that happen?

Reply:
When a user defines ServiceRoleBinding object, which is independent of ServiceRole. See the example of ServiceRoleBinding in rbac.proto. RoleRef is included here for completeness of the proto. 

Does the user interaction happen in control plane only? Is there a case an Envoy is deployed for a service without a control plane? In that case, a user may want to enter ServiceRole and ServiceRoleBinding independently, and the underlying tooling associate a binding to a role based on RoleRef. 

I think @mattklein123 asked the same question.